src/OneExplorer.ts

```diff
@@ -105,67 +107,72 @@ export class OneTreeDataProvider implements vscode.TreeDataProvider<OneNode> {
     const node: Node = {
       type: NodeType.directory,
       name: path.parse(rootPath.fsPath).base,
+      path: rootPath.fsPath,
       childNodes: [],
       uri: rootPath
     };
-    this.searchNode(node, path.dirname(rootPath.fsPath));
+    this.searchNode(node);
     return node;
   }
 
-  private searchNode(node: Node, dirPath: string) {
-    const dirpath = path.join(dirPath, node.name);
+  private searchNode(node: Node) {
+    assert(node.type === NodeType.directory);
+
+    const dirpath = node.path;
     const files = fs.readdirSync(dirpath);
 
-    for (const fn of files) {
-      const fpath = path.join(dirpath, fn);
+    for (const fname of files) {
+      const fpath = path.join(dirpath, fname);
       const fstat = fs.statSync(fpath);
 
       if (fstat.isDirectory()) {
         const dirNode: Node =
-            {type: NodeType.directory, name: fn, childNodes: [], uri: vscode.Uri.file(fpath)};
+            {type: NodeType.directory, name: fname, path: fpath, childNodes: [], uri: vscode.Uri.file(fpath)};
 
-        this.searchNode(dirNode, dirpath);
+        this.searchNode(dirNode);
         if (dirNode.childNodes.length > 0) {
           node.childNodes.push(dirNode);
         }
-      } else if (
-          fstat.isFile() &&
-          (fn.endsWith('.pb') || fn.endsWith('.tflite') || fn.endsWith('.onnx'))) {
-        const modelNode:
-            Node = {type: NodeType.model, name: fn, childNodes: [], uri: vscode.Uri.file(fpath)};
+      } 
+      else if 
+      (fstat.isFile() &&
+        (fname.endsWith('.pb') || fname.endsWith('.tflite') || fname.endsWith('.onnx'))) {
 
-        this.searchPairConfig(modelNode, dirpath);
+        const modelNode:
+          Node = {type: NodeType.model, name: fname, path: fpath, childNodes: [], uri: vscode.Uri.file(fpath)};
 
+        this.searchPairConfig(modelNode);
         node.childNodes.push(modelNode);
       }
     }
   }
 
   /**
    * Search .cfg files in the same directory of the node
```

Comment:
Coudl you add TODO like, `TODO support cfg in different directory` ? 
That is because future reader might be confused with L151, thinking 'is it mandatory to keep tflite and cfg in a same directory? or will this be fixed later?'

Reply:
Sure :-D